views/js/qtiCreator/plugins/button.tpl

```diff
@@ -1,3 +1,3 @@
-<li class="btn-info small {{cssClass}}" title="{{title}}">
+<li class="btn-info small {{cssClass}}" title="{{title}}" {{#if testId}}data-testid="{{testId}}"{{/if}}>
```

Comment:
```suggestion
<li class="btn-info small {{cssClass}}" title="{{title}}"{{#if testId}} data-testid="{{testId}}"{{/if}}>
```

Reply:
Do we have any code style rules for *.tpl files?

Reply:
it is just not to have empty space in the end in case no `data-testid` attribute